index.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+
+const db = require('./data/dbHelpers.js');
+
+const server = express();
+
+server.use(express.json())
+server.use(cors())
+
+const PORT = 4400;
+
+server.get('/api/users', (req, res) => {
+    db.getUsers()
+        .then(users => {
+            res.json(users)
+        })
+        .catch(err => {
+            res.send(err)
```

Comment:
You should send an explicit status here; I'm pretty sure this defaults to `200` (OK) when you're probably wanting a `500` (server error)